### PR TITLE
packit: release: unset use_cockpit by sedding the specfile in packit script

### PR DESCRIPTION
**Pull request by KKoukiou**
Also, remove the unpackaged anaconda-webui files in the case `use_cockpit` is unset.

### CODE REVIEW

File: .packit.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ actions:
   post-upstream-clone:
     - ./autogen.sh
     - ./configure
+    - sed -i 's/use_cockpit\ 1/use_cockpit 0/g' anaconda.spec
```
Comment (jkonecny12) [inline .packit.yml:9]: Honestly I think we should have "make release-cockpit" to solve this better way.

**Review by jkonecny12 — APPROVED** [review]
Other than the minor note below, it looks good to me.

File: .packit.yml (hunk 1 above)
Comment (KKoukiou) [reply .packit.yml:9]: Hmm, can your elaborate? I am not sure I understand.

Comment (KKoukiou) [reply .packit.yml:9]: discussion IRC

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks for fixing this a bit tricky thing! :)

**Pull request merged**